sdk/eventhubs/microsoft-azure-eventhubs/pom.xml

```diff
@@ -76,7 +76,7 @@
     <dependency>
       <groupId>com.microsoft.azure</groupId>
       <artifactId>msal4j</artifactId>
-      <version>0.5.0-preview</version> <!-- {x-version-update;com.microsoft.azure:msal4j;external_dependency} -->
+      <version>1.3.0</version> <!-- {x-version-update;com.microsoft.azure:msal4j;external_dependency} -->
```

Comment:
This is track 1 event hubs module. Did you intend to update msal version of track 1 module too? This might need service team's validation.

Reply:
Seems to be only used in tests. And I don't know a way to specify a dependency of 2 different versions across our SDKs. 

+ @conniey  @mssfang for their opinion

Reply:
Is this because you are using the script to update the external dependency version and it's updating both track 1 and track 2 dependencies?